pages/reviewer/ReviewerHandler.inc.php

```diff
@@ -95,6 +95,24 @@ function _validateAccessKey($request) {
 			$this->user = $user;
 		}
 	}
+
+	/**
+	 * @copydoc PKPReviewerHandler::getReviewForm()
+	 */
+	public function getReviewForm($step, $request, $reviewerSubmission, $reviewAssignment) {
```

Comment:
This should only contain the `case 3`, I think, and the class should be `ReviewerReviewStep3Form`, not `PKPReviewerReviewStep3Form`.

Reply:
ah yes.

If I just add case 3 then I should probably call for the parent function as well. I figured that almost the same amount of code if I just add all three cases?

Reply:
I agree, not a big deal either way; I'm slightly in favour of just overriding the one we care about in the application.

Reply:
how about now?